client/src/pages/EditAccount.js

```diff
@@ -0,0 +1,226 @@
+import React from "react";
+import { useForm, Controller } from "react-hook-form";
+import Checkbox from "../components/Input/Checkbox";
+import SubmitButton from "../components/Button/SubmitButton";
+import styled from "styled-components";
+import FormInput from "../components/Input/FormInput";
+import Title from "../components/Title/Title";
+import UnderLineDescription from "../components/Input/UnderlineDescription";
+const Label = styled.label` 
+  color: ${(props) => props.inputColor || "#425AF2"};
+  padding-left: ${(props) => props.paddingLeft || ""};
+  margin-top: ${(props) => props.marginTop || "1.5rem"};
+  font-size: ${(props) => props.size || ""};
+  font-weight: ${(props) => props.weight || ""};
+};
+`;
+
+export default function EditAccount(props) {
+  // dummy data props,context, redux etc
+  const firstName = "Cees";
```

Comment:
ok I guess before merging you could replace this with the mapped props from redux user

Reply:
@CeesWang Regarding your question from Slack - you could store dummy data like this in Redux and then connect this component with Redux via `mapStateToProps` to consume the different props/dummy data from redux